doc/remediation-flow.md

```diff
@@ -139,7 +123,7 @@ spec:
 ### The scan-remediate-repeat flow
 The general flow is common for the cluster scan as well as the node
 scan. How the remediations are represented and therefore how they are
-applied then differs for node scans and then cluster scan.
+applied then differs for node scans and then platform scan.
```

Comment:
This reads a little strangely, we might say "How the remediations are represented and therefore how they are applied differs for node and platform scans."

Reply:
fixed.